### DISCUSSION TITLE
Javascript error on start

### DISCUSSION THREAD

**Discussion by fotografpaul** [massCodeIO/massCode#99] [Q&A]
Hi!
I get this error when trying to start version 2.6.1. Any ideas how to sort this ?

[https://1drv.ms/u/s!Av4SrNpKu8WshIgRJu3CsTKH09MQNg](url)

Kindest Regards
Paul

**Comment by sangyuxiaowu** [comment]
If you are using windows platform you can try this temporary solution.

https://github.com/massCodeIO/massCode/discussions/156
